.github/workflows/distcheck.yml

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2000 - 2022 Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# SPDX-License-Identifier: curl
+
+name: dist
+
+on:
+  push:
+    branches:
+    - master
+    - '*/ci'
+  pull_request:
+    branches:
+    - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  maketgz-and-verify-in-tree:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - uses: actions/checkout@v3
+
```

Comment:
```suggestion

  - run: sudo apt-get purge -y curl libcurl4 libcurl4-doc
    name: 'remove preinstalled curl libcurl4{-doc}'
```

Reply:
Thanks, but even this does not solve the problem.

Reply:
```
$ find /usr -name "libcurl*"
$ libconfig -v | less
```